service/src/main/kotlin/fi/espoo/evaka/shared/controllers/ExceptionHandler.kt

```diff
@@ -116,26 +106,19 @@ class ExceptionHandler : ResponseEntityExceptionHandler() {
             logger.warn("ClientAbortException ($ex)")
             return null
         }
-        val message = "Unexpected error (${ex.message})"
-        logger.error(message, ex)
+        logger.error("IOException", ex)
         return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body(
-            ErrorResponse(
-                message = ex.message ?: ""
-            )
+            ErrorResponse()
         )
     }
 
     @ExceptionHandler(value = [Throwable::class])
     fun unexpectedError(req: HttpServletRequest, ex: Throwable): ResponseEntity<ErrorResponse> {
         // Checked exceptions get wrapped in UndeclaredThrowableException in kotlin
```

Comment:
This comment no longer makes sense since the code its referring to has been removed